appveyor.yml

```diff
@@ -8,49 +8,32 @@ environment:
     # Avoid long paths on Windows
     STACK_ROOT: "c:\\s"
     STACK_WORK: ".w"
-    CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
+    CACHE_S3_VERSION: v0.1.0
```

Comment:
There is newer release of `cache-s3`: `v0.1.1`, that has an additional feature `--public`, which has no effect cardano-sl, so leaving it at `v0.1.0` is fine as well.